composer.json

```diff
@@ -17,13 +17,14 @@
     "require": {
         "php":                  ">=5.3.6",
         "behat/mink":           "^1.7.1@dev",
-        "symfony/browser-kit":  "~2.3|~3.0",
-        "symfony/dom-crawler":  "~2.3|~3.0"
+        "symfony/browser-kit":  "~2.3|~3.0 || ^4.0",
+        "symfony/dom-crawler":  "~2.3|~3.0 || ^4.0"
     },
 
     "require-dev": {
         "mink/driver-testsuite": "dev-master",
-        "symfony/http-kernel": "~2.3|~3.0"
+        "symfony/http-kernel": "~2.3|~3.0 || ^4.0",
+        "phpunit/phpunit": "^5.7"
```

Comment:
not good, this prevents running CI on PHP 5.3, which is still supported and so must run. Should be `^4.8.35 || ^5.7` instead.

And this is totally useless if you don't change the travis.yml file to use `vendor/bin/phpunit`